src/PlayerOptions.cpp

```diff
@@ -1991,6 +2038,47 @@ class LunaPlayerOptions: public Luna<PlayerOptions>
 
 	FLOAT_NO_SPEED_INTERFACE(VisualDelay, VisualDelay, true);
 
+	static int DisableTimingWindow(T* p, lua_State* L)
+	{
+		int original_top= lua_gettop(L);
+		if (original_top >= 1 && !lua_isnil(L, 1))
+		{
+			// Insert the specified TimingWindow into the disabled windows set.
+			p->m_twDisabledWindows.insert(Enum::Check<TimingWindow>(L, 1));
+		}
+		// Construct a new table indicating all of the disabled windows.
+		lua_newtable( L );
+		for (TimingWindow window : p->m_twDisabledWindows)
+		{
+			Enum::Push(L, window);
```

Comment:
I think there is a `lua_rawseti()` missing.

Reply:
Done. Tbh I'm a bit unfamiliar with the Lua binding things so I might make some mistakes.